src/restapi/fBindings.ts

```diff
@@ -1,11 +1,12 @@
-import {Request, Response} from 'express';
+import {AppCallResponse} from 'mattermost-redux/types/apps';
 
 import {getBindings} from '../bindings';
 import {CtxExpandedActingUserOauth2AppOauth2User} from '../types/apps';
-import {newOKCallResponseWithData} from '../utils/call_responses';
+import {newOKCallResponseWithData, CallResponseHandler} from '../utils/call_responses';
 
-export function fBindings(req: Request, res: Response): void {
+export const fBindings: CallResponseHandler = async (req, res) => {
     const context: CtxExpandedActingUserOauth2AppOauth2User = req.body.context;
     const bindings = getBindings(context);
-    res.json(newOKCallResponseWithData(bindings));
-}
+    const callResponse: AppCallResponse = newOKCallResponseWithData(bindings);
+    res.json(callResponse);
```

Comment:
might be a good idea to be consistent in variable naming (sometimes we use `resp`, most often `res` ) - might be good to see what we use in the webapp code as well.

Reply:
Interesting data here.  FYI, all the code in this repo uses `res` as the response variable name.

In mattermost-webapp the number of occurrences of `res` vs `resp` are listed below.
* 8 - `rg 'resp\.' | wc -l`
* 664 - `rg 'res\.' | wc -l`
* 121 - `rg 'res\.' Gi -v "res.'" Gi "res.\"" Gi "res\.$" Gi "res\. " Gi "features" Gi "Features" G 'res\.' | wc -l`

Reply:
I guess `res` it is then.